RELEASING.md

```diff
@@ -28,7 +28,13 @@ We also assume that ongoing work done is being merged directly to the `master` b
 
 4.  Bump the crate version in `Cargo.toml` to whatever was decided in step 2 for `subxt-metadata`, `subxt-cli`, `subxt-codegen`, `subxt-examples`, `subxt-macro` ,`subxt`, `integration-tests`, `test-runtime`, `ui-tests`.
 
-5.  Update `CHANGELOG.md` to reflect the difference between this release and the last. If you're unsure of
+5.  Ensure the `Cargo.lock` file is up to date.
+
+    ```
+    cargo generate-lockfile
```

Comment:
> This command will create the Cargo.lock lockfile for the current package or workspace. If the lockfile already exists, it will be rebuilt with the latest available version of every package.

should we really bump all dependencies to the latest on each release?
I guess our tests are sufficient but I'm a little bit scared bumping all.

Reply:
bump to latest compatible with substrate.

Reply:
> should we really bump all dependencies to the latest on each release?
> I guess our tests are sufficient but I'm a little bit scared bumping all.

Ah I didn't realise that also bumped dependencies. I guess I'm not too worried overall since it'll just be thel ate3st that our toml files are asking for anyway, which is what a fresh install of everything would bring in?